build.sbt

```diff
@@ -113,7 +108,7 @@ lazy val extras = project
   .settings(sonatypeSettings: _*)
   .settings(
     name := "sbt-jmh-extras",
+    scalaVersion := "2.12.2",
```

Comment:
It should still be possible to use a custom JDK for the forks with `jmh:run -jvm /jdk/1.6`, right?